data/sql/updates/pending_db_world/Scarlet_Preachers_path.sql

```diff
@@ -0,0 +1,91 @@
+(12917000, 4, 1389.8, -5704.65, 136.087, NULL, 0, 0, 0, 100, 0),
+(12917000, 5, 1397.4, -5711.28, 135.041, NULL, 0, 0, 0, 100, 0),
+(12917000, 6, 1406.94, -5723.55, 132.408, NULL, 0, 0, 0, 100, 0),
+(12917000, 7, 1412.76, -5732.12, 131.701, NULL, 0, 0, 0, 100, 0),
+(12917000, 8, 1418.01, -5739.84, 131.22, NULL, 0, 0, 0, 100, 0),
+(12917000, 9, 1422.96, -5747.23, 131.208, NULL, 0, 0, 0, 100, 0),
+(12917000, 10, 1416.89, -5737.62, 131.321, NULL, 0, 0, 0, 100, 0),
+(12917000, 11, 1410.12, -5727, 131.99, NULL, 0, 0, 0, 100, 0),
+(12917000, 12, 1403.86, -5716.92, 133.217, NULL, 0, 0, 0, 100, 0),
+(12917000, 13, 1396.99, -5710.09, 135.191, NULL, 0, 0, 0, 100, 0),
+(12917000, 14, 1389.82, -5704.98, 136.077, NULL, 0, 0, 0, 100, 0),
+(12917000, 15, 1385.97, -5702.27, 138.001, NULL, 0, 0, 0, 100, 0),
+(12917000, 16, 1376.98, -5695.79, 137.968, NULL, 0, 0, 0, 100, 0),
+(12917000, 17, 1367.02, -5688.62, 137.968, NULL, 0, 0, 0, 100, 0);
+
+
+DELETE FROM `waypoint_data` WHERE `id` = 12917100;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES
+(12917100, 1, 1397.56, -5710.58, 135.075, NULL, 0, 0, 0, 100, 0),
+(12917100, 2, 1406.09, -5721.5, 132.634, NULL, 0, 0, 0, 100, 0),
+(12917100, 3, 1415.09, -5735.3, 131.516, NULL, 0, 0, 0, 100, 0),
+(12917100, 4, 1421.83, -5746.68, 131.207, NULL, 0, 0, 0, 100, 0),
+(12917100, 5, 1414.7, -5734.86, 131.553, NULL, 0, 0, 0, 100, 0),
+(12917100, 6, 1406.42, -5721.23, 132.611, NULL, 0, 0, 0, 100, 0),
+(12917100, 7, 1397.92, -5711.19, 134.959, NULL, 0, 0, 0, 100, 0),
+(12917100, 8, 1390.23, -5705.32, 136.028, NULL, 0, 0, 0, 100, 0),
+(12917100, 9, 1386.07, -5702.3, 138.003, NULL, 0, 0, 0, 100, 0),
+(12917100, 10, 1375.49, -5694.62, 137.967, NULL, 0, 0, 0, 100, 0),
+(12917100, 11, 1367.02, -5688.56, 137.967, NULL, 0, 0, 0, 100, 0),
+(12917100, 12, 1377.48, -5695.92, 137.967, NULL, 0, 0, 0, 100, 0),
+(12917100, 13, 1385.91, -5701.93, 137.997, NULL, 0, 0, 0, 100, 0),
+(12917100, 14, 1389.99, -5705.07, 136.058, NULL, 0, 0, 0, 100, 0);
+
+UPDATE `creature` SET `position_x` = 1389.9873, `position_y` = -5705.0527, `position_z` = 136.05995 WHERE `guid` = 129171 AND `id1` = 28594;
+
+UPDATE `creature` SET `MovementType`= 2 WHERE `guid` = 129165 AND `id1` = 28594;
+UPDATE `creature` SET `MovementType`= 2 WHERE `guid` = 129168 AND `id1` = 28594;
+UPDATE `creature` SET `MovementType`= 2 WHERE `guid` = 129170 AND `id1` = 28594;
+UPDATE `creature` SET `MovementType`= 2 WHERE `guid` = 129171 AND `id1` = 28594;
+
+
+DELETE FROM `creature_addon` WHERE (`guid` IN (129165, 129168, 129170, 129171));
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
+(129165, 12916500, 0, 0, 0, 0, 0, NULL),
+(129168, 12916800, 0, 0, 0, 0, 0, NULL),
+(129170, 12917000, 0, 0, 0, 0, 0, NULL),
+(129171, 12917100, 0, 0, 0, 0, 0, NULL);
```

Comment:
These two do not exist on Cata Classic, at least not in the same phase as the others I've observed.

Reply:
I cannot enter on Cata classic PTR (servers are off-line). 
Currently I can only confirm that they are present on retail:

![World Of Warcraft Screenshot 2024 10 15 - 14 24 07 81](https://github.com/user-attachments/assets/2c9792ec-a499-4705-a2c1-b72ad42872b4)

Reply:
Edit: they are also present on Cata Classic: [Video](https://www.youtube.com/watch?v=4teUyejpjjw) from 1:40.

Reply:
It is a phase issue, I was in the initial assault on New Avalon phase, they seem to only be in this Eye of Acherus phase.

Reply:
On Ac these Preachers already are in Eye of Acherus phase.

Reply:
Oh, ok. Now it is clear for me :)

Reply:
https://gist.github.com/heyitsbench/cc972bf4ae11eefa3605a10cd2d0a049
Here's the sniffed data. It appears the two patrolling in front of the chapel are in a formation, i.e. one in front, one following. In the sniffed data, the leader is 0x28594A (fake GUID), and the follower is 0x28594B. My first time even looking at creature formations, so give me a sec to figure that one out. 😛

Reply:
> https://gist.github.com/heyitsbench/cc972bf4ae11eefa3605a10cd2d0a049 Here's the sniffed data. It appears the two patrolling in front of the chapel are in a formation, i.e. one in front, one following. In the sniffed data, the leader is 0x28594A (fake GUID), and the follower is 0x28594B. My first time even looking at creature formations, so give me a sec to figure that one out. 😛

Eh, same for me. I'm currently learning creature formation for another group inside New Avalon (Eye of Acherus phase), but i'm blocked XD